modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -334,7 +335,7 @@ spec:
   configuration:
     velero:
       defaultPlugins:
-        - kubevirt <2>
+        - kubevirt <2> // correction required
```

Comment:
```suggestion
        - aws <2> // correction required
```

Reply:
@amastbau && @weshayutin - before i make this change, please can i ask for a double-check:

```
spec:
  configuration:
    velero:
      defaultPlugins:
        - kubevirt <2> // correction required
        - gcp <3>
        .....

<2> The `kubevirt` plugin is mandatory for OpenShift Virtualization.
<3> Specify the plugin for the backup provider, for example, `gcp`, if it exists.

I am not convinced this is write for VirtDocs